test/test_supervisor.rb

```diff
@@ -213,6 +214,52 @@ def test_main_process_signal_handlers
     $log.out.reset if $log && $log.out && $log.out.respond_to?(:reset)
   end
 
+  def test_cont_in_main_process_signal_handlers
+    omit "Windows cannot handle signals" if Fluent.windows?
+
+    opts = Fluent::Supervisor.default_options
+    sv = Fluent::Supervisor.new(opts)
+    sv.send(:install_main_process_signal_handlers)
+
+    begin
+      Process.kill :CONT, $$
```

Comment:
`Process.pid` is preferred than `$$`  for readability.

```suggestion
      Process.kill :CONT, Process.pid
```